week-2/implement/bmi.js

```diff
@@ -13,3 +13,13 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+function bodyMassIndex(height, weight) {
+  const sum = weight / (height * height);
+  const roundSum = sum.toFixed(1);
+  return roundSum;
+}
```

Comment:
Pro points - can you update this function so it would _also_ work for lbs and inches?